### PR TITLE
Fixed inconsistent use of tabs and spaces in indentation.

### PR DESCRIPTION
**Pull request by ColinKinloch**
Build fails on linux with 'TabError: inconsistent use of tabs and spaces in indentation'

### CODE REVIEW

File: platform/windows/detect.py [hunk 1]
```diff
@@ -188,8 +188,8 @@ def configure(env):
         else:
             VC_PATH = ""
 
-	if (env["openmp"]):
-	    env.Append(CPPFLAGS=['/openmp'])
+    if (env["openmp"]):
```
Comment (akien-mga) [inline platform/windows/detect.py]: That's incorrect, it should have one more indentation level.

Comment (akien-mga) [reply platform/windows/detect.py]: It also needs to be fixed for MinGW line 270.

Comment (akien-mga) [reply platform/windows/detect.py]: And for X11: https://github.com/godotengine/godot/commit/f3ad14224e27f2a25196575e3c43ebc792c90894#diff-99b6145f7696fa9bc930c3667bea7478R268

Comment (ColinKinloch) [reply platform/windows/detect.py]: Sorry, I jumped the gun, it should be correct now.

Comment (akien-mga) [reply platform/windows/detect.py]: It's still wrong, you need to add 4 spaces to all lines you changed for windows.

Comment (Noshyaar) [reply platform/windows/detect.py]: Line 191-192 should be one indentation more.

Comment (ColinKinloch) [reply platform/windows/detect.py]: Right, sorry I should have tested a windows build.

Comment (ColinKinloch) [reply platform/windows/detect.py]: My windows build is failing due to '_finite' and '_isnan' not being declared for thekla_atlas. So inconclusive.

Comment (akien-mga) [reply platform/windows/detect.py]: Yeah but that's another issue. Doesn't change the fact that the indentation here is clearly wrong :)
Just fix the indentation and it's good to go, missing headers to fix windows build can be added in another PR.

Comment (ColinKinloch) [reply platform/windows/detect.py]: Oh, sorry, I fixed the windows indentation but didn't push it correctly.
I meant that considering I didn't notice that the indentation was wrong it's difficult to tell if the corrections actually build or whether my system is just not set up correctly for a mingw build.

**Pull request merged**